cmd/snap/inhibit.go

```diff
@@ -0,0 +1,164 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
```

Comment:
I think it's unlikely this file was originally written in 2016 :smile:

Reply:
Yeah, that's a copy-paste.

Reply:
Fixed.